src/map/map.ts

```diff
@@ -678,13 +678,13 @@ export class PropertiesMap {
         // make a copy of the default layout
         const layout = JSON.parse(JSON.stringify(DEFAULT_LAYOUT)) as typeof DEFAULT_LAYOUT;
         // and set values speific to the displayed dataset
-        layout.xaxis.title = this._options.x.property.value;
-        layout.yaxis.title = this._options.y.property.value;
+        layout.xaxis.title = this._title(this._options.x.property.value);
+        layout.yaxis.title = this._title(this._options.y.property.value);
         layout.xaxis.type = this._options.x.scale.value;
         layout.yaxis.type = this._options.y.scale.value;
-        layout.scene.xaxis.title = this._options.x.property.value;
-        layout.scene.yaxis.title = this._options.y.property.value;
-        layout.scene.zaxis.title = this._options.z.property.value;
+        layout.scene.xaxis.title = this._title(this._options.x.property.value);
+        layout.scene.yaxis.title = this._title(this._options.y.property.value);
+        layout.scene.zaxis.title = this._options.z.property.value; // using this._title(this._options.z.property.value) here seems to cause troubles
         layout.coloraxis.colorscale = this._options.colorScale();
         layout.coloraxis.cmin = this._options.color.min.value;
         layout.coloraxis.cmax = this._options.color.max.value;
```

Comment:
We may also want to display the unit in the color bar title, so below in `layout.coloraxis.colorbar.title.text`; and the in callbacks. What do you think?

Reply:
I think it may look a bit crowded but it is added for now